lib/auth/index.js

```diff
@@ -0,0 +1,50 @@
+import request from 'request';
+import passport from 'passport';
+import OAuth2Strategy from 'passport-oauth2';
+import { authenticateOrLogin, localUser } from './middleware';
+
+const {
+  GRAVITY_API_URL,
+  ARTSY_ID,
```

Comment:
These two should be `GRAVITY_ID` and `GRAVITY_SECRET` to match what we currently have: https://github.com/artsy/metaphysics/blob/e656eebe302aa6e6e4f21607a8bbcca05a824aee/index.js#L24